### PR TITLE
Implement -v & --version version flags and log message when it's doing npm install

### LINKED ISSUE #36: Implement `npub -v` and `npub --version`

### LINKED ISSUE #37: npub looks like it's hanging at the start...
But it's really just doing its thing with node_modules.

It always feels disconcerting when I start it and see it do nothing.  And I guess new users will think there's something wrong (when there isn't).

There should be a message to tell the user what npub is doing.

### PR DESCRIPTION
**Pull request by khoomeister**
Fixes #36
Fixes #37

### CODE REVIEW

File: src/cli.coffee [hunk 1]
```diff
@@ -24,6 +25,11 @@ publishVersion = (str, currentVersion) ->
   return version
 
 cli = (argv, directory, packageJson) ->
+  if argv.v or argv.version
+    npubPackageJson = require path.join(__dirname, '..', 'package.json')
```
Comment (jkrems) [inline src/cli.coffee]: I think a relative require would be enough here.

Comment (khoomeister) [inline src/cli.coffee]: Haha yeah, I implemented it that way first, but then I worried for no good reason at all.

File: src/cli.coffee [hunk 2]
```diff
@@ -1,6 +1,7 @@
 log = require './log'
 npub = require './index'
 minimist = require 'minimist'
+path = require 'path'
```
Comment (EndangeredMassa) [inline src/cli.coffee]: This is no longer required

File: src/publish/npm.coffee [hunk 3]
```diff
@@ -22,6 +22,7 @@ module.exports = (dir, log) ->
 
   install: (registry, callback) ->
     debug "install"
+    log 'removing node_modules & running npm install...'
```
Comment (EndangeredMassa) [inline src/publish/npm.coffee]: I think we should simplify this to `"installing node modules..."`

**Comment by EndangeredMassa** [comment]
In general, I think this should be two different PRs. I don't really mind here, but they two changes should at least be two separate commits.

**Comment by khoomeister** [comment]
All sounds good @EndangeredMassa, changes have been made.

**Comment by EndangeredMassa** [comment]
published in [2.1.0](https://github.com/EndangeredMassa/npub/releases/tag/v2.1.0)

**Pull request merged**